### PR TITLE
Configure versioneer.

### PR DESCRIPTION
**Pull request by wdzhou**
Configure versioneer to fix ORNL task [#110](https://code.ornl.gov/sns-hfir-scse/reflectometry/reflectometry/-/issues/110)

### CODE REVIEW

File: _version.py [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+def get_config():
+    """Create, populate and return the VersioneerConfig() object."""
+    # these strings are filled in when 'setup.py versioneer' creates
+    # _version.py
+    cfg = VersioneerConfig()
+    cfg.VCS = "git"
+    cfg.style = "pep440"
+    cfg.tag_prefix = "v"
+    cfg.parentdir_prefix = "None"
+    cfg.versionfile_source = "datsans/_version.py"
```
Comment (jmborr) [inline _version.py:46]: change `datsans` to `RefRed`

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,10 @@
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = drtsans/_version.py
```
Comment (jmborr) [inline setup.cfg:4]: change `drtsans` to `RefRed`

**Review by jmborr — CHANGES_REQUESTED** [review]

**Comment by jmborr** [comment]
@wdzhou After you make these two little changes, go ahead and merge the PR yourself if you want to

File: _version.py (hunk 1 above)
Comment (wdzhou) [reply _version.py:46]: Done!

**Comment by wdzhou** [comment]
@jmborr Can you review again?  I don't have the permission to merge.

**Pull request merged**